hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -179,5 +179,74 @@
       tests: []
       bazel: []
     },
+    {
+      name: chip_sw_entropy_src_fw_observe_many_contiguous
+      desc: '''Verify that the software can observe samples fast enough to extract many contiguous samples.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode (FIXME: is it necessary to use FIPS mode?).
+            - Enable Firmware Override - Observe.
+            - Collect 1000 samples from the observe FIFO:
+              - Verify after reading each sample that the observe FIFO overflow status has not been set.
+              - FIXME: should the SW run a health check to make sure that it is fast enough?
```

Comment:
The test needs to do multiple such 1000 sample windows. Between windows, we should let Observe FIFO fill up. Then we must completely empty the Observe FIFO and discard the samples. Once it's empty, it will again start collecting. This way we guarantee to collect a 1000 contiguous sample window.